controller/service.rst

```diff
@@ -83,6 +81,74 @@ Then you can define it as a service as follows:
 
         $container->register('app.hello_controller', HelloController::class);
 
+Using the controller shortcuts
+------------------------------
+
+To use the traditional controller shortcuts, you can use the
+:class:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller` class if you
+need to access the container or the
+:class:`Symfony\\Bundle\\FrameworkBundle\\Controller\\AbstractController` class
+if you prefer explicit injection (you can't access the container).
+
+The example above could be revamped to::
+
+    // src/AppBundle/Controller/HelloController.php
+    namespace AppBundle\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+    class HelloController extends AbstractController
+    {
+        public function indexAction($name)
+        {
+            // renders app/Resources/views/hello/index.html.twig
+            return $this->render('hello/index.html.twig', array('name' => $name));
+        }
+    }
+
+With the following configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/services.yml
+        services:
+            _instanceof:
+                Symfony\Bundle\FrameworkBundle\Controller\AbstractController:
```

Comment:
In fact, there is no config to do at all, an AbstractController doesn't need to be a service at all